api/envoy/config/filter/http/grpc_stats/v2alpha/config.proto

```diff
@@ -18,3 +18,12 @@ message FilterConfig {
   // counts.
   bool emit_filter_state = 1;
 }
+
+// gRPC statistics filter state object in protobuf form.
```

Comment:
Should this be linked to the docs at config_http_filters_grpc_stats in a more explicit way? I.e. those docs should explain what is produced.

Reply:
There is a link to arch overview in this file. Is it not sufficient? I think it's mostly for reference documentation. I haven't see other metadata/filter state documented in the intro parts of the docs.

Reply:
I'm talking about a reverse link from the arch overview explaining that this filter offers a proto filter state. It's not a big deal either way to me, but I think as an overall style for docs, it would be nice to have better ways to map easily from extension <-> docs and also extension <-> filter state.

Reply:
Fair enough. Updated the arch docs to cross reference the filter config and the filter object. It almost looks like we need a template for each filter documenting its:
- config proto/versions
- internal filter name
- filter state object